pom.xml

```diff
@@ -1481,11 +1481,33 @@
       </dependency>
 
       <!-- Video -->
+      <dependency>
+        <groupId>org.nuxeo.ecm.platform</groupId>
+        <artifactId>nuxeo-platform-video-parent</artifactId>
+        <version>${nuxeo.platform.version}</version>
+      </dependency>
```

Comment:
Parent modules don't need to be referenced there.

Reply:
Indeed, can you please remove it?